graviti/schema/builtin.py

```diff
@@ -174,6 +226,8 @@ class float64(PortexNumericType):  # pylint: disable=invalid-name
 
     """
 
+    _pyarrow_type_factory = pa.float64
```

Comment:
Will it be better if we make this callable class variable to a method?
Will it be more intuitive and we can use a verb to naming it rather than a noun？